server/routes/authentication.js

```diff
@@ -0,0 +1,48 @@
+const router = require('express').Router();
+const bcrypt = require('bcrypt');
+const Employee = require('../models/Employees');
+const ValidateID = require('./helpers/validateIDCard');
```

Comment:
This should be properly cased:
```suggestion
const validateID = require('./helpers/validateIDCard');
```